src/scout_apm/celery.py

```diff
@@ -1,17 +1,36 @@
 # coding=utf-8
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from celery.signals import task_postrun, task_prerun
+import datetime as dt
+
+from celery.signals import before_task_publish, task_postrun, task_prerun
 
 import scout_apm.core
+from scout_apm.compat import datetime_to_timestamp
 from scout_apm.core.tracked_request import TrackedRequest
 
 
+def before_publish_callback(headers=None, properties=None, **kwargs):
+    if "scout_task_start" not in headers:
+        headers["scout_task_start"] = datetime_to_timestamp(dt.datetime.utcnow())
+
+
 def prerun_callback(task=None, **kwargs):
     tracked_request = TrackedRequest.instance()
     tracked_request.mark_real_request()
 
     span = tracked_request.start_span(operation=("Job/" + task.name))
+
+    start = getattr(task.request, "scout_task_start", None)
+    if start is not None:
+        now = datetime_to_timestamp(dt.datetime.utcnow())
+        try:
+            queue_time = now - start
+        except TypeError:
+            pass
+        else:
+            span.tag("queue_time", queue_time)
```

Comment:
I'm guessing this should be a request tag (aka context) instead of a span tag @dlanderson / @cschneid ? same as discussion on the other celery PR https://github.com/scoutapp/scout_apm_python/pull/205/files#r300129980

Reply:
Yes, a request tag is better here. Especially since it's not describing "the section of code that is running", it's saying something about the request as a whole.